module-2/backend/week_three.md

```diff
@@ -10,14 +10,58 @@ Note: When you're done, submit a PR with a reflection in the comments about how
 ### Questions
 
 1. What is the entry at the command line to create a new rails app?
+rails new
 2. What do Models generally inherit from in rails?
+ApplicationRecord
 3. What do Controllers generally inherit from in a rails project?
+ApplicationController
 4. How would I create a route if I wanted to see a specific horse in my routes fitle assuming I'm sticking to standard conventions and that I didn't want other CRUD functionality?
+Within the routes file, include "resources :horse, only: :show"
 5. What rake task is useful when looking at routes, and what information does it give you?
+rake routes is AMAZING. It gives you the prefix that you would use when referencing a path, the HTTP verb, the URI pattern and the Controller#Action (controller method). 
 6. What is an example of a route helper? When would you use them?
+hmmm horses_path or horse_path(horse) ? You would use them in lieu of using an href path. 
 7. What's the difference between what `_url` and `_path` return when combined with a routes prefix?
+Path is going to start from the root '/' of your directory, when url is going to include an http:// address? 
```

Comment:
Big guess here. Would you be able to / would you want to ever use _url over _path when linking to something within the restful routes? Is path limited to restful routes and url used for other routes?